fastlane/spec/helper/xcodes_helper_spec.rb

```diff
@@ -0,0 +1,66 @@
+describe Fastlane::Helper::XcodesHelper do
+  describe "#read_xcode_version_file" do
```

Comment:
This is more of a curiosity, but, what is the role of `#`?

Reply:
>use the Ruby documentation convention of . (or ::) when referring to a class method's name and # when referring to an instance method's name.

Source: https://www.betterspecs.org

(I learn these things as I go 😆 Ruby is not my mother tongue, clearly haha)

Reply:
🤔 Which made me think... `read_xcode_version_file` is actually a class method, not an instance one. I should probably fix this lol

Reply:
Done :)